lms/static/scripts/browser_check/.babelrc

```diff
@@ -4,6 +4,7 @@
 // This baseline here is based on browsers that support `<script type="module">`.
 {
   "presets": [
+    "@babel/preset-typescript",
```

Comment:
`index.ts` in this directory was being parsed as JavaScript. That happened to work because, until now, it didn't actually contain any TS-specific syntax.